person.rb

```diff
@@ -0,0 +1,31 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # accessor get method
+  attr_reader :id
+
+  # accessor get and set method
+  attr_accessor :name, :age
+
+  # Public method
+  def can_use_services?
+    return true if of_age? || @parent_permission
+
+    false
+  end
```

Comment:
- [Optional] This method will return true if person is of age or if they have obtain permission from the parents. This can simply be written as:

```ruby
    def can_use_services? 
         of_age?() || @parent_permission
    end
```
Can still work like this.